jenkins-pipelines/perf-regression-latency-1TB-ebs-gp3-volumes.jenkinsfile

```diff
@@ -10,5 +10,5 @@ perfRegressionParallelPipeline(
     test_config: '''["test-cases/performance/perf-regression-latency-1TB.yaml", "configurations/ebs/ebs-gp3-2v-4tb-16k-iops.yaml"]''',
     sub_tests: ["test_latency"],
 
-    timeout: [time: 500, unit: "MINUTES"]
+    timeout: [time: 600, unit: "MINUTES"]
```

Comment:
Is this parameter still relevant?
I thought we said that only the test_duration is the relevant parameter now.

Reply:
yes, those should be removed

Reply:
Yep. rechecked the pipeline code. this parameter is not needed any more. Remove it from jenkins files